src/OnboardingSPA/components/Sidebar/components/SitegenEditorPatterns/Sidebar.js

```diff
@@ -128,80 +128,82 @@ const SitegenEditorPatternsSidebar = () => {
 	}, [ currentData ] );
 
 	useEffect( () => {
-		setActiveTab( {
-			name: 'all_versions',
-			title: (
-				<div className="nfd-onboarding-sidebar--sitegen-editor-patterns__header__tab-panel__versions-tab">
-					<p>{ __( 'All Versions', 'wp-module-onboarding' ) }</p>
-				</div>
-			),
-			content:
-				homepages &&
-				activeHomepage &&
-				globalStyles &&
-				Object.keys( homepages ).map( ( homepage, idx ) => {
-					const data = homepages[ homepage ];
-					const newPreviewSettings = cloneDeep( globalStyles[ 0 ] );
-					newPreviewSettings.settings.color.palette =
-						data.color.palette;
-					let blockGrammar = '';
-					[ 'header', 'content', 'footer' ].forEach( ( part ) => {
-						if ( part in data ) {
-							blockGrammar += data[ part ];
-						}
-					} );
-					return (
-						<div
-							className={ `nfd-onboarding-sidebar--sitegen-editor-patterns__header__tab-panel__versions-tab__preview-container` }
-							key={ data.slug }
-						>
+		if ( ! ( activeTab?.home === 'favorites' ) ) {
```

Comment:
```suggestion
		if ( ! ( activeTab?.name === 'favorites' ) ) {
```
I don't think `activeTab?.home` exists.

Reply:
we do have that is defining which tab we are on.